### DISCUSSION TITLE
Time interpretation

### DISCUSSION THREAD

**Discussion by ghost** [community/community#44681] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

In history(log) or on commits there you have a time "committed 5 days ago".
But no "committed 1 days ago" it's called "yesterday" but:
On Jan 18, 2023 is committed Jan 16, 2023, 11:45 PM GMT+1 not yesterday.
It's more than 1 days ago, this will be OK but not yesterday!

**Comment by ghost** [comment]
I changed to "Display current local time"!!
Looks better now, but i'm mostly not logged in...

**Comment by florensvb** [comment]
Came here looking for a similar issue before posting mine:

I have a repo with commits from Jan 20, 2022. Today is Feb 12, 2023, which is roughly one year ago, but it reads "2 years ago":

<img width="1272" alt="Screenshot 2023-02-12 at 13 22 53" src="https://user-images.githubusercontent.com/7792052/218326654-db3e919a-1ed2-4e15-adb6-3b7e586ed497.png">

+1 for making the timestamp-to-text-interpreter a little bit smarter.
